benchmark/opperf/nd_operations/nn_conv_operators.py

```diff
@@ -135,3 +135,50 @@ def run_convolution_operators_benchmarks(ctx=mx.cpu(), dtype='float32', warmup=2
     # Prepare combined results
     mx_conv_op_results = merge_map_list(conv1d_benchmark_res + conv2d_benchmark_res)
     return mx_conv_op_results
+
+
+def run_transpose_convolution_operators_benchmarks(ctx=mx.cpu(), dtype='float32', warmup=10, runs=50):
+    # Conv1DTranspose Benchmarks
+    conv1d_transpose_benchmark_res = []
+    for conv_data in [(32, 3, 256), (32, 3, 64)]:
+        conv1d_transpose_benchmark_res += run_performance_test([getattr(MX_OP_MODULE, "Deconvolution")],
+                                                               run_backward=True,
+                                                               dtype=dtype,
+                                                               ctx=ctx,
+                                                               inputs=[{"data": conv_data,
+                                                                        "weight": (3, 64, 3),
+                                                                        "bias": (64,),
+                                                                        "kernel": (3,),
+                                                                        "stride": (1,),
```

Comment:
These parameters such as stride, dilate and pad seem to make impact to performance. Would it make sense to test those variations?

Reply:
Agreed, for now, I used configuration used in standard architecture to get started. I was planning to cover more variations in next phase with possible automation around input generation. Is that okay?